src/components/layout.js

```diff
@@ -0,0 +1,50 @@
+import React from "react"
+import { useStaticQuery, Link, graphql } from "gatsby"
+import styles from "./styles/layout.module.css";
+
+export default function Layout(props) {
+
+    const data = useStaticQuery(
+        graphql
+        `
+        query {
+            site {
+            siteMetadata {
+                title
+            }
+            }
```

Comment:
Hmm, seems to be an extra } here?

Reply:
Oh, nevermind — I'm misreading. This is the kind of thing that indentation helps with. Ideally, when writing {, [, or (, in JS you want to either keep the brackets on the same line, or break them into multiple lines while lining up the start/close:

```js
// Same-line
sendRequest('/api/signin', values)

foo = [ bar ]

baz: { result { data } } 

query { site { siteMetadata { title } } }
```

That last example is kinda hard to read, because you have to count the brackets to understand what's going on. When you write these as multiple lines, you want to make sure that the opening and closing characters line up:

```js
// Multi-line
sendRequest(
  '/api/signin', 
  values,
)

foo = [ 
  bar,
  baz,
  null,
]

baz: { 
  result { 
    data 
  } 
} 

// A lot simpler to read, because the indentation is suggesting the structure
query { 
  site { 
    siteMetadata { 
      title 
    } 
  } 
}
```